src/app/scheduler/scheduled-process/scheduled-process.component.ts

```diff
@@ -7,6 +7,7 @@ import { CreateScheduleComponent } from '../create-schedule/create-schedule.comp
 import { NgProgress } from 'ngx-progressbar';
 
 enum weekDays {
+  Select = 0,
```

Comment:
can not we avoid this or we used this for direct binding?

Reply:
this is affecting create schedule too, so let's remove it and it does not make sense in `weekDays`

Also in Dropdown default should be `None` instead of `Select`